flloat/base.py

```diff
@@ -246,3 +215,47 @@ def truth(self, tr: FiniteTrace, start: int = 0, end: int = 0) -> bool:
         :param end: the end index.
         :return: True if the regex is satisfied by the trace, False otherwise.
         """
+
+
+class FiniteTraceWrapper(Wrapper, PropositionalTruth, FiniteTraceTruth):
+    """Allow to valuate propositional sentences on finite traces.
+
```

Comment:
With this wrapper, I've adapted propositional sentences to the FiniteTraceTruth interface.
This also solves the problem: `tt` at end is True, but `true` is false, as any other propositional.